src/StreamTransport.php

```diff
@@ -59,6 +59,10 @@ public function send($request)
         try {
             $context = stream_context_create($contextOptions);
             $stream = fopen($url, 'rb', false, $context);
+            if (false === $stream) {
```

Comment:
Would we even reach here? Yii2 error handler will convert error into exception so `fopen` will raise an exception immediately. Have you tested it well?